saleor/graphql/api.py

```diff
@@ -199,6 +230,15 @@ def resolve_order(self, info, id):
     def resolve_orders(self, info, query=None, **kwargs):
         return resolve_orders(info, query)
 
+    def resolve_payment(self, info, id):
+        return get_node(info, id, only_type=Payment)
+
+    def resolve_payment_client_token(self, info, gateway=None):
+        return resolve_payment_client_token(gateway)
+
+    def resolve_payments(self, info, query=None, **kwargs):
+        return resolve_payment_methods(info, query)
+
```

Comment:
Add simple tests to ensure queries are working, especially for `resolve_payment_client_token`, as it looks a little unusual.